pkg/names/names.go

```diff
@@ -31,3 +36,37 @@ const SERVICE_CA_CONFIGMAP = "openshift-service-ca"
 // MULTUS_VALIDATING_WEBHOOK is the name of the ValidatingWebhookConfiguration for multus-admission-controller
 // that is used in multus admission controller deployment
 const MULTUS_VALIDATING_WEBHOOK = "multus.openshift.io"
+
+// TRUST_BUNDLE_CONFIGMAP_KEY is the name of the data key containing
+// the PEM encoded trust bundle.
+const TRUST_BUNDLE_CONFIGMAP_KEY = "ca-bundle.crt"
+
+// TRUST_BUNDLE_CONFIGMAP is the name of the ConfigMap
+// containing the combined user/system trust bundle.
+const TRUST_BUNDLE_CONFIGMAP = "trusted-ca-bundle"
+
+// TRUST_BUNDLE_CONFIGMAP_NS is the namespace that hosts the
+// ADDL_TRUST_BUNDLE_CONFIGMAP and TRUST_BUNDLE_CONFIGMAP
+// ConfigMaps.
+const TRUST_BUNDLE_CONFIGMAP_NS = "openshift-config-managed"
+
+// TRUST_BUNDLE_CONFIGMAP_ANNOTATION is the name of the annotation that
+// determines whether or not to inject the combined ca certificate
+const TRUST_BUNDLE_CONFIGMAP_ANNOTATION = "config.openshift.io/inject-trusted-cabundle"
+
+// Proxy returns the namespaced name of the proxy
+// object named "cluster" in namespace "openshift-config-managed".
+func Proxy() types.NamespacedName {
+	return types.NamespacedName{
+		Name: PROXY_CONFIG,
```

Comment:
Missing the namespace?

Reply:
proxy config is a cluster scoped resource, it is not namespaced.

Reply:
@JacobTanenbaum this but should be fixed when you rebase this PR off #245 that just merged.